### ISSUE TITLE
Partially downloaded file in group of files shown as 100% [found source of bug]

### ISSUE THREAD

**Issue by ethaniel** [feross/webtorrent-desktop#1045]
I'm on 0.17.1 and I'm downloading a torrent with 3 video files. 2 were downloaded fully, 1 was downloaded partially. The file in question is S02E03 AVI. Now it's shown at 100% and there is nothing I can do about it. 

![image](https://cloud.githubusercontent.com/assets/941973/19409556/337834ea-92e7-11e6-80af-f9d0661a1f61.png)

**Comment by ethaniel** [comment]
I think I found the source of the problem. I deleted S02E03 and restart WebTorrent. It started rehashing the torrent. The rehash for S02E02 finished at 99%, when it started to redownload S02E03. 
<img width="743" alt="2016-10-15 21 26 52" src="https://cloud.githubusercontent.com/assets/941973/19412106/76e65a8a-931f-11e6-8b72-8d7431fc2d73.png">

When S02E03 has finished downloaded completely, S02E02 changed from 99% to 100% as well.

![image](https://cloud.githubusercontent.com/assets/941973/19412111/92d20456-931f-11e6-8932-e84e690e1cfd.png)

Since I was downloading series one by one S02E01, S02E02 and then S02E03. When I finished downloading S02E02 it also downloaded the last part of S02E03 (even though I did not finish downloading it). The program thought that I downloaded S02E03 completely as well and resulted in a broken file.

Hope this helps. And I can explain in more details if you didn't understand.
